### PR TITLE
v6x: fix IMU rotations

### PR DESCRIPTION
**Pull request by baumanta**

### CODE REVIEW

**Comment by baumanta** [comment]
@bkueng

**Comment by julianoes** [comment]
@baumanta oh wow, really? Which hardware did you find and test this on?

**Comment by baumanta** [comment]
@julianoes the board types V6X009010 and V6X010010 correspond to new Auterion boards which require these orientations

**Comment by julianoes** [comment]
Oh, I see, that's additions!

**Review by julianoes — DISMISSED** [review]

File: boards/px4/fmu-v6x/init/rc.board_sensors [hunk 1]
```diff
@@ -55,20 +55,29 @@ else
...
 fi
 
 # Internal SPI bus ICM42688p
 if ver hwtypecmp V6X009010 V6X010010
 then
-	icm42688p -R 4 -s start
+	icm42688p -R 12 -s start
 else
-	icm42688p -R 6 -s start
+  if ver hwtypecmp V6X000010
```
Comment (dagar) [inline boards/px4/fmu-v6x/init/rc.board_sensors:76]: Can you fix the whitespace here so it isn't a mix of tabs and spaces?

**Review by bkueng — APPROVED** [review]

**Pull request merged**